### ISSUE TITLE
Add relevant links in the About modal

### ISSUE THREAD

**Issue by nclm** [roundcube/roundcubemail#8395]
When clicking on “About” in Roundcube, the user gets a modal starting like this:

> ### Roundcube Webmail 1.5.1
> 
> Copyright © 2005-2021, The Roundcube Dev Team
> 
> This program is free software; you can redistribute it and/or modify it under the terms of the [GNU General Public License](http://www.gnu.org/licenses/gpl.html) as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
> Some [exceptions](https://roundcube.net/license) for skins & plugins apply.

I believe it really misses a link to Roundcube.

Other improvements could be writing a line about what Roundcube is (after all, if its the default solution provided by my e-mail host, I might except something more informative than technical when clicking on “About”), and possibly to the Github repo.

For instance it could be something like this:

> ### Roundcube Webmail 1.5.1
> 
> This webmail is powered by [Roundcube](https://roundcube.net), a fully-featured browser-based e-mail client.
> 
> Roundcube is free software. You can [contribute to making it better](https://roundcube.net/contribute/) and you can redistribute it and/or modify it under the terms of the [GNU General Public License](http://www.gnu.org/licenses/gpl.html) as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. Some [exceptions](https://roundcube.net/license) for skins & plugins apply. 
> 
> Copyright © 2005-2021, The Roundcube Dev Team

**Comment by thomascube** [comment]
I agree that the wording of the about dialog could be improved and also help reducing support requests from users to us.

**Comment by Cwpute** [comment]
Additionally, a link to the official documentation would be very welcome !

**Comment by nclm** [comment]
I made a PR here: #9720

Things can be rephrased of course, and I would reorder things to have the paragraphs being actual paragraphs (instead of <br/>s) and have the copyright notice very last, but didn’t know how to do that.

**Comment by nclm** [comment]
Great point, I updated :)

**Comment by alecpl** [comment]
Adding a link to Roundcube will not "help reducing support requests from users to us", quite the opposite.

**Comment by Cwpute** [comment]
Good point @alecpl , too many issues are not dependant on RC per se but on the specific installation supplied to end users.
Maybe then, remove the contribution link and only link the main RC website https://roundcube.net/ 
I don't know if that's technically feasible and easy to configure for instance maintainers, but the first lines could instead provide a link to their own support team/documentation instead, who would then be able to redirect it to RC if problems do come from the software and not its configuration.

**Comment by Cwpute** [comment]
I still find this issue relevant, unless changes were made in a recent build which were not linked to this issue ?
